api/app/Http/Controllers/AuthController.php

```diff
@@ -83,6 +83,12 @@ public function authCallback(Request $request)
         $config = $this->fastSigner;
 
         assert($config instanceof Configuration);
+
+        throw_unless(
+            $idToken && is_string($idToken),
+            new InvalidArgumentException('id token is a '.gettype($idToken))
+        );
```

Comment:
Wasn't sure if outputting the type is really needed, and if just pointing out it isn't a string sufficies